### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by Cal-Hagner**
Move documentation to Read The Docs (https://readthedocs.org/) and clean it up.

Also update the package.json license to match the actual license:
- package.json specified MIT
- This repository uses GPL-3.0

### CODE REVIEW

**Comment by Cal-Hagner** [comment]
Once merged, this will need to be implemented by linking the organization to Read the Docs and then importing the repository. Depending on how you choose to name the imported projects, the links in README.md may require updates.

A sample of how it will look is here: https://calhag-novelcovid.readthedocs.io/en/latest/.

File: README.md [hunk 1]
```diff
@@ -1,94 +1,20 @@
...
-|  GET Request  | Output  |
-| ------------ | ------------ |
-|  https://corona.lmao.ninja/all | Returns all total cases, recovery, and deaths. |
-|  https://corona.lmao.ninja/countries | Returns data of all countries that has COVID-19 |
-|  https://corona.lmao.ninja/countries?sort={parameter} | Returns data of each country sorted by the parameter |
-|  https://corona.lmao.ninja/countries/{country-name} | Returns data of a specific country. If an exact name match is desired pass ?strict=true in the query string |
-|  https://corona.lmao.ninja/states | Returns all United States of America and their Corona data |
-|  https://corona.lmao.ninja/jhucsse | **DEPRECATED USE V2 ENDPOINT** Return data from the Johns Hopkins CSSE Data Repository (Provinces and such) |
-| https://corona.lmao.ninja/historical | **DEPRECATED USE V2 ENDPOINT** Get historical data from the start of 2020. (JHU CSSE GISand Data) |
-| https://corona.lmao.ninja/historical/{country-name} | **DEPRECATED USE V2 ENDPOINT** Get historical data from the start of 2020 for a specific country. (JHU CSSE GISand Data) |
```
Comment (ebwinters) [inline README.md]: I think these should stay in README for quick access by people just wanting to use the endpoint, which is separate from the npm package.

Comment (Cal-Hagner) [reply README.md]: I totally understand! I'll get that committed back to normal by tonight.

**Comment by ebwinters** [comment]
@Cal-Hagner one more request, for 
|  GET Request  | Output  |
| ------------ | ------------ |
| https://corona.lmao.ninja/countries?sort={parameter} | Returns data of each country sorted by the parameter |
can you change to: 
|  GET Request  | Output  |
| ------------ | ------------ |
| https://corona.lmao.ninja/countries?sort={parameter} | Returns data of each country sorted by the parameter (`deaths`,`cases`, `recovered`) |

**Comment by ebwinters** [comment]
I want @apollyon600 to check this out since it's his API whose docs will mostly be affected by this. It's debatable if we even need a READTHEDOCS or not @dicedtomatoreal @EliteDaMyth

**Comment by EliteDaMyth** [comment]
@ebwinters FYI i made the api and own it :P apollo is manager in the server

**Comment by EliteDaMyth** [comment]
also this PR seems to be conflicting. can anyone review these changes/approve?

**Comment by ebwinters** [comment]
oh lol, @EliteDaMyth If you think moving docs to readthedocs is a good idea I can resolve and merge ez

**Comment by EliteDaMyth** [comment]
I was thinking maybe something like swagger for the api so people can actually test this shit out. can people interact with the api in readthedocs?

**Comment by ebwinters** [comment]
> Once merged, this will need to be implemented by linking the organization to Read the Docs and then importing the repository. Depending on how you choose to name the imported projects, the links in README.md may require updates.
> 
> A sample of how it will look is here: https://calhag-novelcovid.readthedocs.io/en/latest/.

@EliteDaMyth check it out here, don't think so. Could open swagger readme issue and someone could presumably pick it up

**Comment by EliteDaMyth** [comment]
https://calhag-novelcovid.readthedocs.io/en/latest/installation.html this is wrong there should be 1 = not ==

**Comment by EliteDaMyth** [comment]
also the link doesnt seem to be complete yet

**Comment by Cal-Hagner** [comment]
> I was thinking maybe something like swagger for the api so people can actually test this shit out. can people interact with the api in readthedocs?

Swagger is definitely the better option for consumer-side testing. I honestly was just doing something quick and familiar that looks nice, isn't hard to maintain, and is easily navigable. I wasn't sure what documentation features you were looking for, if any.

If you'd like to use swagger, it's probably best for one of you to set up a SwaggerHub for Teams type of deal to get all the nice free features, but I'd be willing to set this up on a Swagger hosted individual documentation off of my fork so you can get an idea of what it would look like?

**Pull request closed without merging**